mlir/include/hugr-mlir/IR/_Docs.td

```diff
@@ -0,0 +1,9 @@
+#ifndef HUGR_MLIR_IR_DOCS_TD
+#define HUGR_MLIR_IR_DOCS_TD
+
+include "hugr-mlir/IR/HugrDialect.td"
+include "hugr-mlir/IR/HugrAttrs.td"
+include "hugr-mlir/IR/HugrTypes.td"
+include "hugr-mlir/IR/HugrOps.td"
```

Comment:
I will get rid of this and just use HugrOps.td
This will transitively include everything else.
Because of pecularities in the documentation generation tool, typeinterfaces must be in a separate file.